### ISSUE TITLE
Can't import googleapis

### ISSUE THREAD

**Issue by ivanjaros** [grpc-ecosystem/grpc-gateway#1698]
I have been using glide to manage vendoring but it failed to get me the third_party/googleapis with `cannot import "."` error. Since i have experienced a weird bug with glide where it created `vendor/vendor` upon `glide get` I though it is a good time to switch to go modules. But I am getting the same problem - go will not download the google apis since there are no go files. There is some weird note about tooling in the readme, which I don't get and don't want to be fiddling with, but i got most of the code downlaoded and i just need this package to import in my proto files so I am asking - how do I get the entire third_party directory to download into my vendor with go modules like before?

**Comment by johanbrandhorst** [comment]
Hi Ivan. Go Modules doesn't support non-go dependencies. Our recommendation is to manually copy the proto files to your repository. Yes this is terrible, welcome to the protobuf dependency management ecosystem.

**Comment by ivanjaros** [comment]
can't you add empty go file so this wont happen?

**Comment by johanbrandhorst** [comment]
No, we're not going to do that. I agree that the protobuf dependency management ecosystem is unacceptably poor, but we're not going to add empty go files into the third_party folders to work around this for the subset of our users that are using go modules to version their proto files. It's a hack both on our end and on go modules end.

Side note; my current employer is working on making this better. Have a read at https://buf.build/docs/roadmap#buf-schema-registry.

**Comment by ivanjaros** [comment]
ah well. i see now there is that little not in the middle of the readme about copyting the proto files manually...could have been a bit more highlighted i guess.

**Comment by johanbrandhorst** [comment]
I'd be very happy to review a PR that helps highlight this crucial but annoying part of the setup for beginners - I think we tried quoting it but maybe it's not having the desired effect. What do you think?

**Comment by ivanjaros** [comment]
well, the thing is that when i used glide, i got all the code so i did not even know there was this issue to begin with. now that i am trying modules i thought i broke something but it turned out it was "broken" from the get-go. so i guess i am the example of a unknowing user encountering this for the first time. i get the imports, i have done a lot of pb, i just had the impression something got broken so looking back i tihnk it would help if this would be mentioned in the installation section where it clearly states that the googleapis directory has to be copied manually into the code base and proto links have to be adjusted to this directory....if one is using go modules.

**Comment by ivanjaros** [comment]
..or maye i just haven't done new pb stuff for a while and just forgot some stuff around it :D

**Comment by johanbrandhorst** [comment]
Haha, as I said, I'd be happy to review a PR that would highlight anything that was missing. It's hard for me since I helped write the content to know what would've helped you see it better. If you have the time please do contribute.

**Comment by ivanjaros** [comment]
if someone else finds this issue with the same experience, maybe. for now lets say i was just a drop in the ocean :)
